app/src/main/java/org/stepic/droid/ui/fragments/SettingsFragment.kt

```diff
@@ -32,11 +32,14 @@ class SettingsFragment : FragmentBase(), AllowMobileDataDialogFragment.Callback
 
         nullifyActivityBackground()
 
-        setUpNotificationVibration()
-
-        setUpNotifications()
-
-        setUpSound()
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            notificationContainer.visibility = View.GONE
```

Comment:
Maybe we shouldn't hide this button for some part of users?

Reply:
Right, we can open System Notifications here